src/pages/home/category-item/category-item.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { useStyles } from './category-item.style';
+
+const Categories = ({ categoryName, categoryImage, categoryCode }) => {
```

Comment:
Name of component should match with the name of file

Reply:
Renamed